core/components/atoms/input/Input.tsx

```diff
@@ -206,8 +206,11 @@ export const Input = React.forwardRef<HTMLInputElement, InputProps>((props, forw
   });
 
   const trigger = (
-    <div className={rightIconClass}>
-      <Icon name={'info'} size={sizeMapping[size]} />
+    <div
+      className={rightIconClass} // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
+      tabIndex={0}
+    >
+      <Icon name={'info'} size={sizeMapping[size]} className="Input-rightIcon" />
```

Comment:
We do not need to wrap Icon inside the `div` container

Reply:
The div was already present and without the div, there are changes in color of different icons in different states